modules/thermal_hydraulics/doc/content/modules/thermal_hydraulics/bib/thm.bib

```diff
@@ -83,4 +83,14 @@ @techreport{wolf_mccarthy1960
 institution = {Rocketdyne},
 number = {RR-60-12},
 year = {1960}
-}
\ No newline at end of file
+}
+
+@book{todreas2021nuclear,
```

Comment:
I think you already added it in another PR.

Reply:
Yes, I added in the htc_core branch that got merged, but I think I wrote this file before the merging, so I added it to compile the documentation, I will remove it